nfs/src/rpc_task.cpp

```diff
@@ -1313,30 +1273,49 @@ void rpc_task::run_write()
      * membufs. We don't wait for the writes to actually finish, which means
      * we support buffered writes.
      */
-    std::vector<bytes_chunk> bc_vec =
-            copy_to_cache(this, ino, bufv, offset, length, error_code);
+    uint64_t periodic_bytes = 0;
+
+    copy_to_cache(this, ino, bufv, offset, length, error_code, &extent_left, &extent_right);
```

Comment:
copy_to_cache() needs to copy given data to the given inode's file cache, so it should be an nfs_inode's member function.

Reply:
bufv must be  const ptr since we don't want callee to change that.

Reply:
also since now we don't have a need to return bc vector from copy_to_cache(), we can remove it